colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>This can be on a different or follow-up notebook as per Noa's suggestion to use the core registries + collections</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>